### PR TITLE
More concise puzzles

### PR DESCRIPTION
**Pull request by romey-sklar**
Makes puzzles more concise by:

1) removing duplicates of correct answers in options list
2) cleaning up formatting in puzzlesV2.json to make things look the same and more condensed

### CODE REVIEW

File: app/hooks/useDailyPuzzle.tsx [hunk 1]
```diff
@@ -53,13 +53,25 @@ const useDailyPuzzle: (day: string | null) => Puzzle | null = (day: string | nul
...
+        const allOptions = [...correctOptions, ...incorrectOptions]
+
+        // Ensure options are sorted alphabetically
+        const sortedOptions = sortOptionsByText(allOptions)
+
         setTodayPuzzle({ ...dailyPuzzle, options: sortedOptions })
       } else {
         const optionsList: string[] = options[dailyPuzzle.optionsKey]
 
+        optionsList.sort() // Ensure options are sorted alphabetically
```
Comment (jkvaternik) [inline app/hooks/useDailyPuzzle.tsx:73]: do we need this second sort?

Comment (romey-sklar) [reply app/hooks/useDailyPuzzle.tsx:73]: that's just to ensure all old puzzles have sorted options, but could easilly take it or leave it

Comment (romey-sklar) [reply app/hooks/useDailyPuzzle.tsx]: it's gone

**Review by jkvaternik — APPROVED** [review]
:send-it:

**Pull request merged**